todoApp/spec/main.tsp

```diff
@@ -275,9 +278,18 @@ namespace TodoItems {
 
     @sharedRoute
     @post
-    op createAttachment(
+    op createUrlAttachment(
+      @header contentType: "application/json",
+      @path itemId: TodoItem.id,
+      @body contents: TodoUrlAttachment,
+    ): WithStandardErrors<NoContentResponse | NotFoundResponse>;
+
+   @sharedRoute
+    @post
+    op createFileAttachment(
+      @header contentType: "multipart/form-data",
       @path itemId: TodoItem.id,
-      @body contents: TodoAttachment,
-    ): WithStandardErrors<NoContentResponse | NotFoundErrorResponse>;
+      @body contents: TodoFileAttachment,
+    ): WithStandardErrors<NoContentResponse | NotFoundResponse>;
```

Comment:
I know this is still in draft status but I'd like to clarify if the change is expected from `NotFoundErrorResponse` to `NotFoundResponse`?
See pr for more discussion: https://github.com/allenjzhang/typespec-e2e-demo/pull/19

Reply:
Yeah, that was just a mistake, good catch!